### PR TITLE
chore(sdk-coin-zketh): fix transaction endcoding method for zksync

### PR DESCRIPTION
**Pull request by mukeshsp**
Ticket: WIN-1961

### CODE REVIEW

File: modules/statics/src/networks.ts [hunk 1]
```diff
@@ -963,6 +963,10 @@ class ZkSyncTestnet extends Testnet implements EthereumNetwork {
   chainId = 300;
   nativeCoinOperationHashPrefix = '300';
   tokenOperationHashPrefix = '300-ERC20';
+  forwarderFactoryAddress = '0xdD498702F44c4da08EB9E08D3F015eEfE5cB71Fc';
+  forwarderImplementationAddress = '0xBE69CAE311191Fb45E648ed20847F06faD2DBAB4';
+  walletFactoryAddress = '0x4550e1e7616D3364877FC6c9324938Dab678621a';
+  walletImplementationAddress = '0x92dB2759d1DcA129a0D9d46877f361Be819184C4';
 }
```
Comment (gianchandania) [inline modules/statics/src/networks.ts:970]: Can you please make these addresses all small case

File: modules/sdk-coin-zketh/test/unit/send.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import { TransactionType } from '@bitgo/sdk-core';
+import should from 'should';
+import { decodeTransferData } from '@bitgo/abstract-eth';
```
Comment (gianchandania) [inline modules/sdk-coin-zketh/test/unit/send.ts:3]: I think this needs to be inside transactionBuilder folder

File: modules/statics/src/networks.ts (hunk 1 above)
Comment (mukeshsp) [reply modules/statics/src/networks.ts:970]: updated !!

File: modules/sdk-coin-zketh/test/unit/send.ts (hunk 2 above)
Comment (mukeshsp) [reply modules/sdk-coin-zketh/test/unit/send.ts]: done

**Review by gianchandania — APPROVED** [review]

**Pull request merged**